plugins/filter/jc.py

```diff
@@ -21,6 +21,67 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+DOCUMENTATION = '''
+  name: jc
+  short_description: Convert output of many shell commands and file-types to JSON
+  version_added: 1.1.0
+  author: Kelly Brazil (@kellyjonbrazil)
+  description:
+    - Convert output of many shell commands and file-types to JSON.
+    - Uses the L(jc library,https://github.com/kellyjonbrazil/jc).
```

Comment:
Should this be in a `requirements` key?

Reply:
It is there as well, see further below.

Reply:
ah! I did look for it, but I am not used to seeing the key below options, my mistake!